cli/dcoscli/experimental/main.py

```diff
@@ -71,17 +72,22 @@ def _info():
     return 0
 
 
-def _add(dcos_package):
+def _add(dcos_package, package_name, package_version):
     """
     Adds a DC/OS package to DC/OS
 
     :param dcos_package: path to the DC/OS package
-    :type dcos_package: str
+    :type dcos_package: None | str
+    :param package_name: the name of a remote DC/OS package
+    :type package_name: None | str
+    :param package_version: the version of a remote DC/OS package
+    :type package_version: None | str
     :return: process status
     :rtype: int
     """
     package_manager = get_package_manager()
-    response = package_manager.package_add(dcos_package)
+    response = package_manager.package_add(
+        dcos_package, package_name, package_version)
```

Comment:
Actually, I'm wary of checking that they are not both specified, because the library should take care of that. If at any point they are both specified that would be a programmer error, and I've been told by @tamarrow in the past that I should not code for programmer errors.